core/src/main/scala/matryoshka/instances/fixedpoint/package.scala

```diff
@@ -67,4 +67,18 @@ package object fixedpoint {
         case (_, _)                 => NilF()
       }
   }
+
+  /** Encodes a function that may diverge.
+    */
+  type Partial[A] = Nu[A \/ ?]
+  type ⊥[A] = Partial[A]
```

Comment:
Oh, I see... Yeah, the subscripted `\bot` is standard notation from domain theory for adjoining a point/bottom to a dcpo, and makes sense. I think the Scala notation might be a bit tricky to understand, since you can't do subscripts (somebody get on that, lol!)... But now I see what you were going for!